src/steps/command.ts

```diff
@@ -238,7 +246,7 @@ export class CommandStep extends LabeledStep {
         /* eslint-disable @typescript-eslint/camelcase */
         return {
             ...(await super.toJson(opts)),
-            command: transformCommand(this.command),
+            command: Command.transformCommand(this.command),
```

Comment:
doesn't this need to be:

```suggestion
            command: Command[transformCommandKey](this.command),
```
now? Do the tests still pass with the current changes?